crates/transaction-pool/src/test_util/mock.rs

```diff
@@ -247,14 +247,14 @@ impl MockTransaction {
     /// Returns a new transaction with a higher gas price +1
     pub fn inc_price(&self) -> Self {
         let mut next = self.clone();
-        let gas = self.get_gas_price() + 1;
+        let gas = self.get_gas_price().checked_add(U256::from(1)).unwrap();
```

Comment:
This will panic on overflow now, which was not the case before in debug mode. I guess it's fine?

Reply:
Given it's a `test-util` mock, i think it's fine